src/api_routes_http.rs

```diff
@@ -248,6 +251,7 @@ pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimitCell) {
           .route("/like", web::post().to(like_post))
           .route("/like/list", web::get().to(list_post_likes))
           .route("/save", web::put().to(save_post))
+          // TODO should these be moved into the new report heading?
```

Comment:
Makes sense to have only `/report/list`, I cant think of any reason why you'd want only comment reports specifically (and if so it could be a parameter instead of separate endpoints).

Reply:
Gotcha, I can remove these then.